topi/recipe/gemm/gemm_int8.py

```diff
@@ -137,12 +137,15 @@ def block_size_filter(entity):
     print(task.config_space)
 
     measure_option = autotvm.measure_option(
-        measure_func='local', number=10, n_parallel=8, timeout=20)
+        builder=autotvm.LocalBuilder(),
+        runner=autotvm.LocalRunner(repeat=3, min_repeat_ms=100, timeout=4)
```

Comment:
cc @vinx13 
We found our old measurement on GPU is inaccurate, especially when the kernel is fast (< 1ms).
Use `min_repeat_ms` to make sure the duration of measurement can make it more accurate.